### PR TITLE
 fix Issue 18519: missing curl + phobos = std.net.curl unittest timing out

### PR DESCRIPTION
**Pull request by ibuclaw**
Second attempt at fixing the issue.

Only a little sceptical that this is more correct here - calling shutdown is recommended however in the docs.  In any case, I can no longer reproduce this on Linux after applying patch.

### CODE REVIEW

**Comment by ibuclaw** [comment]
```
std.socket.SocketAcceptException@std/socket.d(2848): Unable to accept socket connection: Bad file descriptor
----------------
****** FAIL debug32 std.net.curl
std.net.curl.CurlException@std/net/curl.d(4203): Failed to load curl, tried "libcurl.so", "libcurl.so.4", "libcurl-gnutls.so.4", "libcurl-nss.so.4", "libcurl.so.3".
----------------
gmake[1]: *** [posix.mak:386: unittest/std/net/curl.run] Error 1
gmake[1]: *** Waiting for unfinished jobs....
```
In the context of the issue, this means PASS.

**Comment by ibuclaw** [comment]
> However, as people like the auto-tester to be green, I'll partially undo the last commit I guess...

Moved disabling of tests to the makefile, rather than the module, which prevented tests from running on FreeBSD regardless of what compiler is building.

File: posix.mak [hunk 1]
```diff
@@ -382,8 +392,14 @@ endif
 moduleName=$(subst /,.,$(1))
 
 # target for batch unittests (using shared phobos library and test_runner)
-unittest/%.run : $(ROOT)/unittest/test_runner
```
[the remark this answers is not in the document]
Comment (ibuclaw) [reply posix.mak]: Ok, so I fixed the disabled tests recipe then.

**Comment by ibuclaw** [comment]
Should be good to go.

**Comment by thewilsonator** [comment]
Doc tester 
```
Cloning into '../dub'...
fatal: unable to access 'https://github.com/dlang/dub/': Could not resolve host: github.com
posix.mak:593: recipe for target '../dub' failed
```

**Comment by wilzbach** [comment]
CC @CyberShadow this doc failure seems to happen more often these days :/

**Comment by CyberShadow** [comment]
> CC @CyberShadow this doc failure seems to happen more often these days :/

Forgot to reply but I think I found & fixed the problem yesterday.

**Comment by thewilsonator** [comment]
Nice, thanks!

**Comment by thewilsonator** [comment]
Nice, thanks!

**Pull request merged**